src/generic/generic-component-file/generic-component-file.twig

```diff
@@ -128,6 +127,7 @@
                   {% include '@ecl/' ~ system ~ '-component-button' with {
                     'label': _button_label ~ '<span class="ecl-u-sr-only">(' ~ translation.meta ~ ')',
                     'modifier': 'secondary',
+                    'type_attribute': 'button',
                     'extra_classes': 'ecl-file__translations-download',
                     'extra_attributes': [{ 'name': 'type', 'value': 'button' }]
```

Comment:
`'extra_attributes': [{ 'name': 'type', 'value': 'button' }]` not needed anymore